alacritty/res/text.f.glsl

```diff
@@ -21,7 +23,7 @@ void main() {
         color = vec4(bg.rgb, 1.0);
     } else if ((int(fg.a) & COLORED) != 0) {
         // Color glyphs, like emojis.
-        vec4 glyphColor = texture(mask, TexCoords);
+        vec4 glyphColor = texture2D(mask, TexCoords);
```

Comment:
Have you tested these new shaders on machines that support OpenGL 3.3+? Did you notice any regressions?

Reply:
tested, no issues or regressions.
texture2D() and texture() are one and the same under the hood. With OpenGL 3+ more capabilities were added to what texture2D() can do, so it was renamed to texture().
The naming texture2D was deprecated and is not available in OpenGL 3.3 core any more, but is still available in all compatibility profiles.

Reply:
That's my understanding too, I'll still run a few framerate benchmarks once I've confirmed this works properly on macOS.

Reply:
Actually, on my Nvidia Graphics card it fails to run, whilst on my Intel OpenGL 3+ Laptop it runs fine. Maybe the NVidia driver does not assume Compatibility by default?
I guess I'll add seperate Shaders based on GL_SHADING_LANGUAGE_VERSION after all...

Reply:
I added a GLSL Version check. Now both my Thinkpad T500 and my GTX 1060 are happy.
Since the original GLSL files are used by default, there should be no change in either performance, or compatibility.